### ISSUE TITLE
Problem not recognizing games in the file in the game location

### ISSUE THREAD

**Issue by TheMercenary333** [yuzu-emu/yuzu-android#9]
If there is a file containing one more game in the file specified as the game location, the games look like in the picture. If transferred directly to the specified location, the problem disappears. Can you bring a fix for this?

![Screenshot_20230531_132834_yuzu.jpg](https://github.com/yuzu-emu/yuzu-android/assets/74915480/ec2b9508-23a9-47df-93e8-efa81e96e6f8)

**Comment by MonsterDruide1** [comment]
I don't quite understand your issue here. What file are you placing where? Could you rephrase the first sentence of your issue please?

**Comment by TheMercenary333** [comment]
> I don't quite understand your issue here. What file are you placing where? Could you rephrase the first sentence of your issue please?

Let's say I set the path /storage/emulated/0/ROMS as a game folder. If there are folders containing games such as "New Ones" in this directory, yuzu does not recognize the games in this file.

![Screenshot_20230604_225440_ZArchiver](https://github.com/yuzu-emu/yuzu-android/assets/74915480/b47a5e94-718d-4eba-9b55-07f542df13d4)

**Comment by MonsterDruide1** [comment]
Alright, so you want ROMs in subdirectories to be parsed and displayed correctly. Got it, let's see what the Yuzu devs say about it.
